emperor/core.py

```diff
@@ -278,6 +291,31 @@ def make_emperor(self, standalone=False):
         # avoid unicode strings
         coord_ids = list(map(str, self.mf.index.tolist()))
 
+        # TODO: This will be removed once the custom axes creation is moved to
+        # the graphical user interface i.e. to the Axes tab.
+        if custom_axes:
```

Comment:
1 => 1 vs. PC1 vs. PC2, 
2 => 1 vs. 2 vs. PC1,
3 => 1 vs. 2 vs. 3 -- we don't need PCs 
4 => 1 vs. 2 vs. 3 -- we don't need PCs and 4 ...